wiki/Rules/Content_usage_permissions/de.md

```diff
@@ -1,50 +1,467 @@
----
-outdated_translation: true
-outdated_since: 866f5bfcb2e396728fd9b942bec87327b1221283
----
-
 # Richtlinien für die Nutzung von Inhalten
 
-Als Mapper solltest du Berechtigungen von relevanten Rechteinhabern für Audio-/visuelle/Gameplay-Inhalte in deine Beatmap-Einreichungen einfügen. Ohne Erlaubnis werden deine Beatmaps möglicherweise entfernt und du läufst Gefahr, dass dein Zugang zum Einreichungssystem nach der [osu!-Urheberrechtspolitik](/wiki/Legal/Copyright) verweigert wird.
+Im Mittelpunkt von osu! steht die Möglichkeit, Kreationen zu entwerfen und mit anderen zu teilen. Es muss sichergestellt werden, dass für die Verwendung des Werkes eines anderen Künstlers sachgemäße Genehmigungen eingeholt wurden, und dass der Urheber ordnungsgemäß angegeben wird.
 
-Um sicherzustellen, dass dein Account in einem guten Ruf bleibt, ist es empfehlenswert, Beatmaps mit Hilfe von Inhalten zu erstellen, die eines der folgenden Dinge sind:
+Ohne Erlaubnis werden hochgeladene [Beatmaps](/wiki/Beatmap) möglicherweise entfernt und dem dafür verantwortlichen Nutzer könnte laut der [osu!-Urheberrechtsrichtlinie](/wiki/Legal/Copyright) der Zugang zum Einreichungssystem verwehrt werden. Daher sollten Nutzer Beatmaps mit Inhalten erstellen, die den folgenden Bedingungen entsprechen:
 
 - als legal zur Verwendung in osu! erklärt
-- zugelassen zur Nutzung von ihrem/ihren Ersteller(n)
+- von ihrem/ihren Ersteller(n) zur Nutzung zugelassen
 - urheberrechtsfrei
 
+## Opt-in
+
+Künstler, die ihren Eintrag in dieser Liste hinzufügen oder ändern möchten, können das osu!-Musiklizenzteam unter [music@ppy.sh](mailto:music@ppy.sh) kontaktieren.
+
+Für die Lizenzvergabe kann man sich ebenfalls an dieses Team wenden. Siehe [osu!-Musiklizenzierung](/wiki/Legal/Music_licensing) für mehr Informationen.
+
 ## Audio
 
-Wir stellen Mappern eine Sammlung an lizenzierten Songs über das [Featured-Artist](/wiki/People/Featured_Artists)-Programm zur Verfügung. Jeder Song in [der Auflistung](https://osu.ppy.sh/beatmaps/artists) ist kostenlos für die Verwendung innerhalb von osu! und birgt kein Risiko, dass er entfernt wird.
+*Hinweis: Das Veröffentlichen oder Streamen von Spielinhalten auf anderen Seiten könnte möglicherweise anderen Lizenzbestimmungen unterliegen. Bitte gehe bei der Liveübertragung von Spielinhalten mit der gebotenen Sorgfalt vor.*
```

Comment:
```suggestion
*Hinweis: Das Veröffentlichen oder Streamen von Spielinhalten auf anderen Seiten könnte möglicherweise anderen Lizenzbestimmungen unterliegen, die zuerst ausgehandelt oder akzeptiert werden müssen. Bitte gehe bei der Liveübertragung von Spielinhalten mit der gebotenen Sorgfalt vor.*
```
I had this in mind first, but I think this can be omitted as the context should be clear.